### ISSUE TITLE
Focused elements - update focus style

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#561]
# Description
In order to improve accessibility, elements focused via tab should be styled to improve visibility. See screenshots.

## Action Items

- [ ] Update focus styles per mockups

## Resources

<img width="560" alt="Screen Shot 2020-04-23 at 5 10 41 PM" src="https://user-images.githubusercontent.com/40484278/80161623-e22d8500-8585-11ea-8d5b-bd84d8196c7e.png">
<img width="568" alt="Screen Shot 2020-04-23 at 5 10 47 PM" src="https://user-images.githubusercontent.com/40484278/80161709-13a65080-8586-11ea-8a91-03a2e6888439.png">
